tests/ui/test_discopane.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from pages.discopane import DiscoveryPane
+
+
+@pytest.fixture(scope='session')
+def base_url(base_url):
+    '''
+    This is hardcoded for now, once we go through about:addons
+    we'll be able to use the template expansion
+    '''
+    base_url += '/en-US/firefox/discovery/pane/48.0/Darwin/normal'
+    return base_url
+
+
+@pytest.fixture
+def firefox_profile(firefox_profile, base_url):
+    '''
+    Set preferences necessary to load discovery pane, and
+    for installing extensions and toggling switches to
+    install and uninstall.
+    '''
+    firefox_profile.set_preference('extensions.webapi.testing', True)
+    firefox_profile.set_preference('extensions.webservice.discoverURL',
+                                   base_url)
+    firefox_profile.update_preferences()
+    return firefox_profile
+
+
+@pytest.fixture
+def discovery_pane(selenium, base_url):
+    return DiscoveryPane(selenium, base_url).open()
+
+
+@pytest.mark.nondestructive
+def test_that_discovery_pane_loads(discovery_pane):
+    assert discovery_pane.is_discopane_visible
+    assert len(discovery_pane.uninstalled_addons) == 7
+
+
+@pytest.mark.nondestructive
+def test_that_welcome_video_plays(discovery_pane):
+    assert discovery_pane.is_video_closed
+    discovery_pane.play_video()
+    assert discovery_pane.is_video_playing
+    discovery_pane.close_video()
+    assert discovery_pane.is_video_closed
```

Comment:
Looking more closely at the values we're checking here, I think we should rename these to more accurately reflect what we're checking. Perhaps `is_close_video_displayed` and `is_play_video_displayed`. The test would then look like:

```python
    assert not discovery_pane.is_close_video_displayed
    discovery_pane.play_video()
    assert not discovery_pane.is_play_video_displayed
    discovery_pane.close_video()
    assert discovery_pane.is_play_video_displayed
    assert not discovery_pane.is_close_video_displayed
```